sympy/vector/parametricregion.py

```diff
@@ -0,0 +1,47 @@
+from sympy.core.basic import Basic
+from sympy.core.containers import Dict, Tuple
+
+class ParametricRegion(Basic):
+    """
+    Represents a parametric region in space.
+
+    Examples
+    ========
+
+    >>> from sympy import symbols, cos, sin, pi
+    >>> from sympy.vector import ParametricRegion
+
+    >>> r, theta = symbols("r theta")
+    >>> circle = ParametricRegion((r, theta), r*cos(theta), r*sin(theta), limits={theta: (0, pi)})
+    """
+    def __new__(cls, parameters, definition, limits, system=None):
```

Comment:
What about setting `limits` to be optional? For example, with a default `limits=None` argument.

Reply:
Yes, It will be better. I will do it.